### PR TITLE
Message source backend

### PR DESCRIPTION
**Pull request by martinraison**
@stkem

### CODE REVIEW

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/eliza/app/com/keepit/eliza/model/Message.scala [hunk 1]
```diff
@@ -68,6 +68,29 @@ object MessageSender {
   }
 }
 
+case class MessageSource(value: String)
+object MessageSource {
+  val CHROME = MessageSource("Chrome")
+  val FIREFOX = MessageSource("Firefox")
```
Comment (2is10) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/model/Message.scala:74]: Do we think users will care which browser their friends are using the extension on?
We may need to add this to our Privacy Policy (no joke).

Comment (stkem) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/model/Message.scala:74]: Showing the actual type of browser (as opposed to just "Browser") was a deliberate decision, basically to advertise  subtly what  platforms we support. Privacy policy is a good point though.

Comment (martinraison) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/model/Message.scala:74]: Another possible behavior UI-wise would to be to only show "via iPhone/iPad/iPod/Android/Email" but not "via Chrome/Firefox". It still leaves the privacy issue open though

**Pull request merged**